### PR TITLE
Add distributed db config

### PR DESCRIPTION
**Pull request by ernestas-poskus**
@migibert 

Hey, adding distributed DB configuration 

> http://orientdb.com/docs/2.0/orientdb.wiki/Distributed-Configuration.html#default-distributed-db-configjson

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -32,6 +32,23 @@ orientdb_distributed:
   tcp_enabled: False
   tcp_members: []
 
+orientdb_distributed_db_config:
```
Comment (ernestas-poskus) [inline defaults/main.yml]: these are defaults

File: templates/default-distributed-db-config.j2 [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "autoDeploy": {{orientdb_distributed_db_config.autoDeploy | to_json }},
+  "hotAlignment": {{orientdb_distributed_db_config.hotAlignment | to_json }},
+  "executionMode": "{{orientdb_distributed_db_config.executionMode}}",
+  "readQuorum": {{orientdb_distributed_db_config.readQuorum}},
+  "writeQuorum": {{orientdb_distributed_db_config.writeQuorum}},
+  "failureAvailableNodesLessQuorum": {{orientdb_distributed_db_config.failureAvailableNodesLessQuorum | to_json}},
+  "readYourWrites": {{orientdb_distributed_db_config.readYourWrites | to_json}},
+  "servers": {{orientdb_distributed_db_config.servers | to_json}},
```
Comment (migibert) [inline templates/default-distributed-db-config.j2]: I do not see the "servers" variable at this level in documentation. I see it as part of cluster dictionary, is it a mistake or did I miss something ?

Comment (ernestas-poskus) [inline templates/default-distributed-db-config.j2]: It is present in documentation 
screenshot from 2016-06-12 00-57-41

you can try to run this playbook without this commit and see that it generates: default-distributed-db-config.json file

Comment (migibert) [inline templates/default-distributed-db-config.j2]: You are right, my bad !

**Comment by migibert** [comment]
Thx for this PR !

**Comment by ernestas-poskus** [comment]
Pleasure

**Pull request merged**